packages/esm-appointments-app/src/hooks/useVisits.ts

```diff
@@ -6,16 +6,15 @@ import SelectedDateContext from './selectedDateContext';
 
 /**
  * Custom hook to fetch visits from the OpenMRS REST API.
+ * This fetches all visits that started on *or after* the currently selected date and are associated with the current session location.
```

Comment:
Note that this useVisit fetches all visits started on *or after* the currently selected date, while I think the intent was for it *just* to fetch visits on the selected date.

From a functional perspective, I don't think this introduced any bugs because the *only* time this is actually used to trigger anything is if the day it "today", but there's a lot of needless loading.  I will ticket cleaning this up.